config/nginx.conf.erb

```diff
@@ -34,10 +34,6 @@ http {
             return 301 'https://micromasters.mit.edu';
         }
 
-        location ^~ /static/(.*$) {
-            try_files $uri $uri/ staticfiles/$1$args staticfiles/$1/$args =404;
-        }
-
```

Comment:
We don't want to remove this because having Nginx serve the static assets is much more efficient than delegating that to uWSGI or Django.